atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/ThreeNodeCassandraClusterOperations.java

```diff
@@ -58,6 +60,10 @@ public void replicateSystemAuthenticationDataOnAllNodes()
         }
     }
 
+    public void disableAutoCompaction() throws IOException, InterruptedException {
+        runNodetoolCommandOnAllContainers("disableautocompaction", NODETOOL_REPAIR_TIMEOUT_SECONDS);
```

Comment:
would suggest a much much smaller timeout. I'd expect this to be faster than `nodetool status`.